docs/api/covidcast-signals/chng.md

```diff
@@ -29,6 +29,8 @@ commercial purposes.
 
 | Signal | Description |
 | --- | --- |
+| `7dav_inpatient_covid` | Ratio of inpatient hospitalizations associated with COVID-19, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time with a trailing 7 day average <br/> **Earliest date available:** 2020-01-01 |
+| `7dav_outpatient_covid` | Ratio of outpatient doctor visits with confirmed COVID-19, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time with a trailing 7 day average <br/> **Earliest date available:** 2020-01-01 |
```

Comment:
If you could add the last available issue that would be nice. Otherwise I think this is good. 
```suggestion
| `7dav_inpatient_covid` | Ratio of inpatient hospitalizations associated with COVID-19, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time with a trailing 7 day average. This historical signal is no longer updated. <br/> **Earliest date available:** 2020-01-01 |
| `7dav_outpatient_covid` | Ratio of outpatient doctor visits with confirmed COVID-19, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time with a trailing 7 day average. This historical signal is no longer updated. <br/> **Earliest date available:** 2020-01-01 |
```